src/EditorFeatures/Core/Wrapping/AbstractCodeActionComputer.cs

```diff
@@ -86,6 +86,29 @@ protected abstract class AbstractCodeActionComputer<TWrapper> : ICodeActionCompu
 
             protected abstract Task<ImmutableArray<WrappingGroup>> ComputeWrappingGroupsAsync();
 
+            protected string GetSmartIndentationAfter(SyntaxNodeOrToken nodeOrToken)
```

Comment:
This is just taking the existing code that exists for this purpose elsewhere, and moves it to a common location so that all wrapping features can use it :)